tools/get_active_unactive_users.js

```diff
@@ -0,0 +1,66 @@
+var conf = require('../loop/config').conf;
+var async = require('async');
+var redis = require("redis");
+
+var storage = conf.get("storage");
+var utils = require("./utils");
+var keysInformation = utils.keysInformation;
+var dbInformation = utils.dbInformation;
+
+var args = process.argv.slice(2);
+
+var verbose = args.indexOf('--verbose') !== -1;
+
+var A_DAY = 3600 * 24;
+var TWO_WEEKS = 3600 * 24 * 7 * 2;
+var A_MONTH = 3600 * 24 * 30;
+
+
+if (storage.engine === "redis") {
+  var options = storage.settings;
+  var client = redis.createClient(
+    options.port,
+    options.host,
+    options.options
+  );
+  if (options.db) client.select(options.db);
+
+  keysInformation(client, 'hawk.*', function(err, keysInfo) {
+    if (err) throw err;
+
+    console.log("Processing " + keysInfo.length + " keys");
+
+    var active = 0;
+    var unactive = 0;
+    var fifteen = 0;
```

Comment:
biweekly is more explicit for non native speakers... 
I saw twice-monthly too :)